### PR TITLE
Favorites functionality

### PR DESCRIPTION
**Pull request by bronsonmullens**
I've created model and model controller files. Tapping the favorite button will now append that city to an array of favorite cities. I also fixed a minor error with the network controller conforming to a nonexistent protocol.

### CODE REVIEW

File: labs-ios-starter/Map/DetailViewController.swift [hunk 1]
```diff
@@ -24,7 +26,18 @@ class DetailViewController: UIViewController {
...
+                favoriteButton.image = UIImage(systemName: "heart.fill")
+                cityController.favoriteToggled(city: city)
+            } else {
+                favoriteButton.image = UIImage(systemName: "heart")
+                if cityController.favoriteCities.contains(city) {
+                    cityController.favoriteToggled(city: city)
+                }
+            }
+        }
     }
```
Comment (alext8900) [inline labs-ios-starter/Map/DetailViewController.swift:42]: Simple yet elegant implementation

File: labs-ios-starter/Model/NetworkController.swift [hunk 2]
```diff
@@ -17,19 +17,14 @@ protocol NetworkPlaceholder {
...
     let network: NetworkPlaceholder
     init(network: NetworkPlaceholder = URLSession.shared) {
         self.network = network
     }
     
-    func capturingUserInput(userInput: String) {
-        myUserInput = userInput
-        print(myUserInput)
-    }
-
```
Comment (alext8900) [inline labs-ios-starter/Model/NetworkController.swift:32]: Thanks for fixing this. Forgot that I was conforming to this protocol that I removed.

**Review by alext8900 — APPROVED** [review]
Good job Bronson, hopefully this should be able to fix our issue we were having.

File: labs-ios-starter/Login/LoginViewController.swift [hunk 3]
```diff
@@ -104,7 +104,6 @@ class LoginViewController: UIViewController {
     }
     
     @objc func didTapLogin() {
-        print("login tapped")
```
Comment (denced) [inline labs-ios-starter/Login/LoginViewController.swift:107]: thanks for deleting this, I forgot last pr

**Review by denced — APPROVED** [review]

**Pull request merged**